lib/storage/file.js

```diff
@@ -1061,7 +1061,19 @@ File.prototype.getSignedPolicy = function(options, callback) {
 
   makeAuthorizedRequest_.getCredentials(function(err, credentials) {
     if (err) {
-      callback(err);
+      var signingError = new Error('Signing failed. See `error` property.');
+      signingError.error = err;
+      callback(signingError);
```

Comment:
IMO if it makes our code base even a little bit cleaner I think it's worth it. I'll open a separate issue